CHANGELOG

```diff
@@ -74,6 +74,14 @@ Improved testing
 
 Miscellaneous
 -------------
+- Added support for explicit vectorization for the high-performance templated
+  code, which can now take advantage of specific intrinsics classes on the
+  following architectures:
+    - x86 (SSE/AVX/AVX2/AVX512),
+    - Power8 & Power9 (VSX),
+    - BG/Q (QPX).
+  It is now enabled by default, and can be disabled with MFEM_USE_SIMD=NO.
```

Comment:
Maybe mention here the `linalg/simd` directory and the updates to the `t*.hpp` sources?